### PR TITLE
feat: add REGION variable

### PR DESCRIPTION
**Pull request by Tchoupinax**
Hello :)

I found your script useful but I needed to use the ```--region``` flag to change the region of my deployment.
I offer this PR to add this feature to your repo

Thanks for your script :)

Have a nice day!

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+**
```
Comment (Tchoupinax) [inline .dockerignore:1]: It is better to have a `.dockerignore` file

File: script.sh [hunk 2]
```diff
@@ -1,8 +1,16 @@
 # stop if vercel token is missing
-[ -z $TOKEN ]&&printf "Token is missing"&&exit 1
+[ -z $TOKEN ] && printf "Token is missing" && exit 1
```
Comment (Tchoupinax) [inline script.sh:2]: I prefer using space for lisibility

File: script.sh [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
+else
+  printf "\nPRODUCTION DEPLOYMENT\n"&&PROD="--prod";
+fi
 
 # run deployment and return url
-URL=$(vercel --confirm --token $TOKEN $PROD)&&printf "\nDEPLOYMENT SUCCESSFUL\n$URL"
+if [ -z $REGION ];then
+  URL=$(vercel --confirm --token $TOKEN $PROD) && printf "\nDEPLOYMENT SUCCESSFUL\n$URL"
+else
+  URL=$(vercel --confirm --regions $REGION --token $TOKEN $PROD) && printf "\nDEPLOYMENT SUCCESSFUL\n$URL"
```
Comment (Tchoupinax) [inline script.sh:15]: I add the flag if the environment variable is detected, maybe there is a better way to do that

**Review by Tchoupinax — COMMENTED** [review]
comments for your understanding

**Comment by maximilianschmelzer** [comment]
Looks good. Thank you for the addition and I'm glad you found it useful.

**Pull request merged**